specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/rbac.json

```diff
@@ -0,0 +1,544 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The delete request was accepted and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleAssignmentResponse"
+            }
+          },
+          "default": {
+            "description": "400 InvalidRoleAssignmentId - The Role Assignment ID '{0}' is not valid. Role Assignment ID must be a GUID."
+          },
+          "204": {
+            "description": "Role Assignment does not exist."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{databaseAccountName}/cosmosRoleAssignments": {
+      "get": {
+        "operationId": "SqlRoleAssignment_List",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleAssignmentList": {
+            "$ref": "./examples/CosmosDBSqlRoleAssignmentList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "description": "Retrieves the list of all Azure Cosmos DB Role Assignments based on the filter.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/filterRoleAssignmentParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The relevant Role Assignments",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleAssignmentListResult"
+            }
+          },
+          "default": {
+            "description": "400 UnsupportedQuery - The filter '{0}' is not supported. Supported filters are either 'atScope()' or 'principalId eq '{value}' or assignedTo('{value}')'."
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SqlRoleDefinitionProperties": {
+      "properties": {
+        "roleName": {
+          "type": "string",
+          "description": "A user-friendly name for the Role Definition. Must be unique for the database account."
+        },
+        "type": {
```

Comment:
I was kind of unclear about this one since in the PDF it was a RoleDefinitionType, but should it be an enum?